packages/hdwallet-seeker/src/adapter.ts

```
@@ -0,0 +1,210 @@
+
+  solanaGetAccountPaths(_msg: SolanaGetAccountPaths): SolanaAccountPath[] {
+    const SOLANA_BIP44_PATH: BIP32Path = [0x80000000 + 44, 0x80000000 + 501, 0x80000000 + 0]
+    return [{ addressNList: SOLANA_BIP44_PATH }]
+  }
+
+  solanaNextAccountPath(_msg: SolanaAccountPath): SolanaAccountPath | undefined {
+    return undefined
+  }
+
+  async solanaSignTx(msg: SolanaSignTx): Promise<SolanaSignedTx | null> {
+    const transaction = solanaBuildTransaction(msg, this.pubkey)
```

ESLint

[error] Insert `;`

(prettier/prettier)